packages/theme/src/lib/legacy-theme.ts

```diff
@@ -508,6 +508,15 @@ export const legacyTheme = {
         maxHeight: '100px',
       },
     },
+    MuiCheckbox: {
+      styleOverrides: {
+        root: {
```

Comment:
looks like legacy doesn't have any padding on the checkbox, idk if it was intentional to leave the default padding so they migrators can see where it's going and don't add a bunch of extra padding during v0.

Reply:
My thought was that more often than not, the padding would be necessary. So I left it. I'm open to a discussion though.